openpype/hosts/traypublisher/plugins/publish/collect_online_file.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+import pyblish.api
+from pathlib import Path
+
+
+class CollectOnlineFile(pyblish.api.InstancePlugin):
+    """Collect online file and retain its file name."""
+    label = "Collect Online File"
+    order = pyblish.api.CollectorOrder
+    families = ["online"]
+    hosts = ["traypublisher"]
+
+    def process(self, instance):
+        file = Path(instance.data["creator_attributes"]["path"])
+
+        if not instance.data.get("representations"):
```

Comment:
Instances created in new publisher should always have prepared `"representations"` key with list.

Reply:
BTW if `"representations"` are filled then this representation won't be created?